src/CONST.ts

```diff
@@ -1488,7 +1488,10 @@ const CONST = {
     FORM_CHARACTER_LIMIT: 50,
     LEGAL_NAMES_CHARACTER_LIMIT: 150,
     LOGIN_CHARACTER_LIMIT: 254,
-    WORKSPACE_NAME_CHARACTER_LIMIT: 80,
+
+    TITLE_CHARACTER_LIMIT: 100,
+    SUPPORTING_CHARACTER_LIMIT: 500,
```

Comment:
I am wondering if this should be `SUPPORTING_TEXT` or `DESCRIPTION`.

@youssef-lr  wdyt?

Reply:
I think `DESCRIPTION_LIMIT` is more consistent with what we have in the backend.

Reply:
done